lib/src/index.ts

```diff
@@ -102,23 +130,17 @@ export async function main() {
   }
 
   if (!useVscode) {
-    const s = spinner();
-    s.start('Removing .vscode folder... ⏳');
     const destPath = path.join(process.cwd(), projectName as string);
     await removeFolder('.vscode', destPath);
-    s.stop('Done ✅');
```

Comment:
I removed this spinner, it was printing 4 spinners in the terminal. Now, we're printing just one at the end of the command.

<img width="1322" alt="image" src="https://github.com/eruptionjs/create-eruption/assets/2914170/2b230d9d-6a09-493f-b272-f30c108cdb96">